builtin/providers/rundeck/resource_project.go

```diff
@@ -0,0 +1,292 @@
+package rundeck
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/apparentlymart/go-rundeck-api/rundeck"
+)
+
+var projectConfigAttributes = map[string]string{
+	"project.name":                          "name",
+	"project.description":                   "description",
+	"service.FileCopier.default.provider":   "default_node_file_copier_plugin",
+	"service.NodeExecutor.default.provider": "default_node_executor_plugin",
+	"project.ssh-authentication":            "ssh_authentication_type",
+	"project.ssh-key-storage-path":          "ssh_key_storage_path",
+	"project.ssh-keypath":                   "ssh_key_file_path",
+}
+
+func resourceRundeckProject() *schema.Resource {
+	return &schema.Resource{
+		Create: CreateProject,
+		Update: UpdateProject,
+		Delete: DeleteProject,
+		Exists: ProjectExists,
+		Read:   ReadProject,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Unique name for the project",
+				ForceNew:    true,
+			},
+
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Description of the project to be shown in the Rundeck UI",
+			},
```

Comment:
Okay... will do. I think in most cases folks will want to provide a value here since it is displayed prominently in the UI, but no harm in providing a default to be consistent with everything else.